package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "auth0": "^2.27.0",
     "auth0-extension-tools": "^1.4.4",
-    "auth0-source-control-extension-tools": "^4.1.12",
+    "auth0-source-control-extension-tools": "~4.3.0",
```

Comment:
Note to maintainers: back in July, I found that setting this matcher to a caret (`^`) can be very risky for existing customers, because adding new features to `auth0-source-control-extension-tools` (semver minor) can be breaking for `auth0-deploy-cli`. Here, I'm reverting back to tilde matching (`~`) to make it safer to deploy CLI consumers in the future.

I don't think there's a simple way to automate a check for this and annoyingly, `npm` defaults to caret. We'll need to rely on code reviews to detect this...

Reply:
BTW, 4.2.0 was never published, that's why the minor version skips to `3`...